tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2667,11 +2691,17 @@ <h2>Show similarity to selected datapoint</h2>
         }
       },
 
-      // TODO (tolgab) Update for model comparison
+      nearestCounterfactualModelChanged_: function() {
+        if (this.showNearestCounterfactual) {
+          this.findClosestCounterfactual_();
```

Comment:
I see a bit of repetition with `nearestCounterfactualDistChanged_`. Would it make sense to just use an observer and do
```ts
observers: [
  ...,
  'maybeFindClosestCounterFactual_(showNearestCounterfactual, nearestCounterfactualModel, nearestCounterfactualDist)'
],
```

Reply:
Good idea, done!